src/commands/command-fns/private-chat.js

```diff
@@ -0,0 +1,106 @@
+const Discord = require('discord.js')
+const {privateChannelPrefix, getSend} = require('../utils')
+
+async function privateChat(message) {
+  const mentionedMembers = Array.from(message.mentions.members.values())
+  const mentionedMembersNicknames = Array.from(
+    message.mentions.members.values(),
+  ).map(m => m.nickname ?? m.user.username)
+  if (mentionedMembers.length < 2) {
+    return message.channel.send(`You should mention at least two members.`)
+  }
+
+  const allPermissions = Object.keys(Discord.Permissions.FLAGS)
+  const membersPermissions = mentionedMembers.map(member => {
+    return {
+      type: 'member',
+      id: member.id,
+      allow: [
+        'ADD_REACTIONS',
+        'VIEW_CHANNEL',
+        'SEND_MESSAGES',
+        'SEND_TTS_MESSAGES',
+        'READ_MESSAGE_HISTORY',
+        'CHANGE_NICKNAME',
+      ],
+    }
+  })
+
+  let channelSuffix
+  if (mentionedMembersNicknames.length === 2) {
+    channelSuffix = mentionedMembersNicknames.join('-')
+  } else {
+    channelSuffix = `${mentionedMembersNicknames
+      .slice(0, 2)
+      .join('-')}-and-others`
+  }
+
+  const everyoneRole = message.guild.roles.cache.find(
+    ({name}) => name === '@everyone',
+  )
+
+  const categoryPrivateChat = message.guild.channels.cache.find(
+    ({name, type}) =>
+      type === 'category' && name.toLowerCase().includes('private chat'),
+  )
+
+  const allActivePrivateChannels = message.guild.channels.cache.filter(
+    channel =>
+      channel.type === 'text' &&
+      channel.parentID === categoryPrivateChat.id &&
+      channel.name.includes('-private-') &&
+      !channel.deleted,
+  )
+
+  const existingChat = allActivePrivateChannels.find(channel => {
+    const chatMembers = channel.members
+      .filter(member => !member.user.bot)
+      .map(member => member.id)
+
+    return (
+      chatMembers.length === mentionedMembers.length &&
+      mentionedMembers.every(member => chatMembers.indexOf(member.id) !== -1)
+    )
+  })
+
+  if (existingChat) {
+    return message.channel.send(`There is already a chat for the same members`)
```

Comment:
Nice feature! Could we reference it?

```suggestion
    return message.channel.send(`There is already a chat for the same members ${existingChat}`)
```